protocol/avdl/chat1/local.avdl

```diff
@@ -256,8 +258,22 @@ protocol local {
     string senderUsername;
     string senderDeviceName;
     string senderDeviceType;
+
+    // MessageBoxed.V1: Hash of the encrypted header ciphertext.
+    // MessageBoxed.V2: Hash of MB.headerSealed (.v || .n || .b)
     Hash headerHash;
+
+    // TOOD Maybe get rid of this field in favor of verificationKey.
+    //      If so, bump-nuke the caches in storage_blockengine and any other persistent
+    //      storage of MessageUnboxedValid.
+    // MessageBoxed.V1: Header signature. Included for the verification key.
+    // MessageBoxed.V2: Null
```

Comment:
Old code wouldn't crash. And 'old' (pre-read-MB.V2) code should never get this far because it can't unbox a MessageBoxedV2.

The only place that `HeaderSignature` value is used so far is here https://github.com/keybase/client/blob/master/go/chat/convsource.go#L375